v1.4.1 fails with gcc-7

```
[  959s] [ 61%] Building CXX object src/libxtp/CMakeFiles/votca_xtp.dir/qmpackages/gaussian.cc.o
[  959s] cd /home/abuild/rpmbuild/BUILD/xtp-1.4.1/build/src/libxtp && /usr/bin/c++  -Dvotca_xtp_EXPORTS -I/home/abuild/rpmbuild/BUILD/xtp-1.4.1/build/include -I/home/abuild/rpmbuild/BUILD/xtp-1.4.1/include -I/home/abuild/rpmbuild/BUILD/xtp-1.4.1/build/src/libxtp  -fomit-frame-pointer -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -DNDEBUG -fopenmp -O1 -fPIC   -std=gnu++11 -o CMakeFiles/votca_xtp.dir/qmpackages/gaussian.cc.o -c /home/abuild/rpmbuild/BUILD/xtp-1.4.1/src/libxtp/qmpackages/gaussian.cc
[  965s] /home/abuild/rpmbuild/BUILD/xtp-1.4.1/src/libxtp/qmpackages/gaussian.cc: In member function 'virtual bool votca::xtp::Gaussian::ParseLogFile(votca::xtp::Orbitals*)':
[  965s] /home/abuild/rpmbuild/BUILD/xtp-1.4.1/src/libxtp/qmpackages/gaussian.cc:1150:43: error: call of overloaded 'advance(std::__cxx11::list<std::__cxx11::basic_string<char> >::iterator&, int)' is ambiguous
[  965s]                      advance(coord_block, 3);
```
Details [here](https://github.com/votca/xtp/files/1376342/_log.txt)

--- a/src/libxtp/qmpackages/gaussian.cc
+++ b/src/libxtp/qmpackages/gaussian.cc
@@ -1147,7 +1147,7 @@ namespace votca {
                     boost::iter_split(stringList, archive, boost::first_finder("\\\\"));
 
                     list<std::string>::iterator coord_block = stringList.begin();
-                    advance(coord_block, 3);
+                    std::advance(coord_block, 3);
 
                     std::vector<std::string> atom_block;
                     boost::algorithm::split(atom_block, *coord_block, boost::is_any_of("\\"), boost::algorithm::token_compress_on);
@@ -1181,7 +1181,7 @@ namespace votca {
                     }
 
                     // get the QM energy out
-                    advance(coord_block, 1);
+                    std::advance(coord_block, 1);
                     std::vector<std::string> block;
                     std::vector<std::string> energy;
                     boost::algorithm::split(block, *coord_block, boost::is_any_of("\\"), boost::algorithm::token_compress_on);

